conf/forbidden_user_names.conf

```diff
@@ -5,17 +5,31 @@
 /\-$/
 /^dev$/
 /^blog$/
+/^support$/
+/^helpdesk$/
 /wikidot/
 /wikijump/
 /^pro$/
 /^web$/
 /^ssl$/
+/^tls$/
 /^payment[s]?$/
 /^pay$/
+/^secure$/
 /^service[s]?$/
-/^admin$/i
+/^guru$/
+/^admin$/
+/^administrator$/
+/^mod$/
 /^moderator$/
+/^anon$/
 /^anonymous$/
+/^guest/
 /^root$/
+/^error$/
+/^null$/
+/^undefined$/
 /^bot$/
-/^support$/
\ No newline at end of file
+/^robot$/
+/^O5-\w+$/
+/^SCP-\w+$/
```

Comment:
Do we want to extend sites the ability to disallow certain nicknames from joining? We could also check renames against site specific configurations at that point as well.

Reply:
I think that makes sense, but would be a separate issue outside the scope of this PR. I was thinking these make sense since, our main Wikijump installation would be for EN, official sandboxes, etc.

Reply:
Alright, I don't want this to block this pull, I think when we've got a bit more new code written we might circle back on this and see if we can pull these site-specific requirements out of the globals.